h/views/client.py

```diff
@@ -39,6 +39,12 @@ def _client_url(request):
     csp_insecure_optout=True,  # This view adds its own custom Content Security Policy
     http_cache=(60 * 5, {"public": True}),
 )
+@view_config(
+    route_name="profile_app",
+    renderer="h:templates/app.html.jinja2",
+    csp_insecure_optout=True,  # This view adds its own custom Content Security Policy
+    http_cache=(60 * 5, {"public": True}),
+)
```

Comment:
This `@view_config(..)` call is the same as for notebook and sidebar, only `route_name` is different. I need to check if there's some way to not triplicate the code.